packages/core/android/src/test/kotlin/com/datadog/tools/unit/forge/DdSdkConfigurationForgeryFactory.kt

```diff
@@ -38,7 +39,13 @@ class DdSdkConfigurationForgeryFactory : ForgeryFactory<DdSdkConfiguration> {
                     null
                 )
             },
-            trackingConsent = forge.aNullable { anElementFrom("pending", "granted", "not_granted") }
+            trackingConsent = forge.aNullable { anElementFrom("pending", "granted", "not_granted") },
+            configurationForTelemetry = ConfigurationForTelemetry(
+                initializationType = forge.anAlphabeticalString(),
```

Comment:
thought: we can wrap each in `forge.aNullable { }` to cover `.?let` branches, but it can make assertions more complex.

Reply:
I believe this is not needed right now, as all the telemetry fields will always be defined, but we cannot type them as such.